### ISSUE TITLE
Hardcoded localhost prevents remote use

### ISSUE THREAD

**Issue by jtuu** [stackgl/shader-school#160]
It appears that some things in the client are hardcoded to connect to "localhost", which means shader-school can't be used from a remote location. More specifically "localhost:12491/changes" is what it's trying to connect to.
This could probably be fixed by creating the url dynamically like this
`var url = 'http://' + window.location.host + '/changes'`

With a quick search I couldn't find anything related to "/changes" in this repo, so I'm not sure if this issue is relevant to shader-school itself.

**Comment by belohlavek** [comment]
Yep, this is a problem for me too, shall I submit a pull request?

**Comment by shichongrui** [comment]
This was being caused by a dependency of shader-school, glslify-live. I have submitted a pull request to the author of that package with the fix. https://github.com/hughsk/glslify-live/pull/6

Once that is merged, because it is not a breaking change it should only warrant a patch version bump and should get picked up by shader-school automatically. A simple npm install -g shader-school again will get the change.
